build/.gitignore

```diff
@@ -1,6 +0,0 @@
-Packager/Assets/Plugins
-Packager/Library
```

Comment:
why did you remove this file?

Reply:
Restored the file. Thought it was blocking the tracking of the .tmpl files but something else was doing it.